src/test/java/org/eolang/jeo/representation/xmir/XmlNodeTest.java

```diff
@@ -93,6 +95,42 @@ void retrievesText() {
         );
     }
 
+    @Test
+    void retrievesChild() {
+        final XmlNode child = new XmlNode(
+            "<program><o>text</o></program>"
+        ).child("o");
+        final String expected = "<o>text</o>";
+        MatcherAssert.assertThat(
+            String.format(
+            "Retrieved XML: %s does not match with expected %s",
+                child,
+                expected
+            ),
+            child,
+            new IsEqual<>(new XmlNode(expected))
+        );
+    }
+
+    @Test
+    void retrievesChildObjects() {
+        final List<XmlNode> objects = new XmlNode(
+            "<program><o>o1</o><o>o2</o></program>"
+        ).children().collect(Collectors.toList());
+        final List<XmlNode> expected = new ListOf<>(
+            new XmlNode("<o>o1</o>"),
+            new XmlNode("<o>o2</o>")
+        );
+        MatcherAssert.assertThat(
+            String.format(
+            "Retrieved child objects: %s don't match with expected: %s",
+                objects,
+                expected
+            ),
+            objects, new IsEqual<>(expected)
```

Comment:
@h1alexbel Maybe it's better to place `new IsEqual<>(expected)` on the next line for consistency with the previous test? What do you think?